storage/nfs-utils/rpc-statd.yaml

```diff
@@ -0,0 +1,114 @@
+name: rpc-statd
+depends:
+  - service: cri
+  - service: ext-rpcbind
+  - network:
+      - addresses
+      - connectivity
+      - hostname
+      - etcfiles
+container:
+  entrypoint: /usr/local/sbin/rpc.statd
+  args:
+    - --foreground
+    - --no-syslog
+  security:
+    maskedPaths: []
+    readonlyPaths: []
+  mounts:
+    # netconfig file required by libtirpc
```

Comment:
if these files doesn't need to be on the host, let's just put it into the container rootfs and stop using the mounts